packages/ibm-products/src/components/Datagrid/Datagrid.stories.jsx

```diff
@@ -36,7 +36,7 @@ import { Wrapper } from './utils/Wrapper';
 import DocsPage from './Datagrid.docs-page';
 import { getBatchActions } from './utils/getBatchActions';
 import { StatusIcon } from '../StatusIcon';
-import { Annotation } from '../../../../core/.storybook/Annotation';
+import { Annotation } from '../../../../ibm-products/.storybook/Annotation';
```

Comment:
I'm seeing a lot of imports where you go all the way out to `/ibm-products` but you can just do `../../../.storybook/Annotation` since you're already in `/ibm-products`.

Reply:
ahh yes. it's so nice that sometimes vscode chooses not to update the paths when you move something 🙄